modules/app/dapr-containerapps-environment/test/containerApp.bicep

```diff
@@ -0,0 +1,151 @@
+@description('Specifies the name of the container app.')
+param containerAppName string = 'containerapp-${uniqueString(resourceGroup().id)}'
+
+@description('Specifies the name of the container app environment.')
+param containerAppEnvName string
+
+@description('Specifies the location for all resources.')
+param location string
+
+@description('Specifies the docker container image to deploy.')
+param containerImage string = 'mcr.microsoft.com/azuredocs/containerapps-helloworld:latest'
+
+@description('Specifies the container port.')
+param targetPort int = 80
+
+@description('Specifies the dapr app port.')
+param daprAppPort int = targetPort
+
+@allowed([
+  'http'
+  'grpc'
+  ''
+])
+param daprAppProtocol string = 'http'
+
+@allowed([
+  'Single'
+  'Multiple'
+])
+param revisionMode string = 'Single'
+
+@description('Number of CPU cores the container can use. Can be with a maximum of two decimals.')
```

Comment:
Can we include some sample values in the description? I've seen several get tripped up that this is a string value that contains a number.

Reply:
disregard - I didn't realize this was a test

Reply:
Thats ok - it's good feedback for #131